types/boxLabel.d.ts

```diff
@@ -0,0 +1,46 @@
+import { Color, FontSpec } from 'chart.js';
+import { PartialEventContext } from './events';
+import { DrawTime, Scriptable } from './options';
+
+export interface BoxLabelOptions {
+	drawTime?: Scriptable<DrawTime, PartialEventContext>,
+	font?: FontSpec
+	color?: Scriptable<Color, PartialEventContext>,
+	/**

```

Comment:
nit: inconsistent style, most comment blocks have an empty line before, and tabs instead of spaces (linter is misconfigured as it does not detect these)

Reply:
Do you have the config to apply to my lint locally? I always run it to check these kinds of issues

Reply:
Fixed in master, rebase to get the new config